### PR TITLE
update dockerfile with figures

### LINKED ISSUE #16: add wesanderson library to dockerfile for r markdowns

### LINKED ISSUE #18: add figure rmds and dependencies to r_demos docker container

### PR DESCRIPTION
**Pull request by allaway**
This contains the paper figure Rmds in the r_demo docker container and dependency wesanderson.

closes #18 and closes #16 

Also referenced in #8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,9 +60,9 @@ indexfile:
 
 ## RNA-seq and Exome-Seq figures
 
-The code to generate the omics figures in the manuscript can be found in the [figs](figs/) directory. Specifically the ExomeSeq markdown can be viewed [here]() and the RNASeq markdown can be viewed [here](). 
+The code to generate the omics figures in the manuscript can be found in the [figs](figs/) directory. Specifically the ExomeSeq markdown can be viewed [here]() and the RNASeq markdown can be viewed [here](). These notebooks are also contained in the R demo notebook container described below.
```
Comment (jaybee84) [inline README.md]: To regenerate the figure panel showing copy number variant analysis, refer to docker_cnv [here](https://github.com/Sage-Bionetworks/JHU-biobank/tree/master/figs/docker_cnv).

Comment (allaway) [reply README.md]: is this docker container deposited anywhere?

Comment (jaybee84) [reply README.md]: in nfosi

Comment (allaway) [reply README.md]: OK, I will link to that instead.

Comment (jaybee84) [reply README.md]: Sounds good :)

Comment (nfosi-service) [reply README.md]: OK, I've updated the docker repo for the cnv docker container to autobuild, so now any changes pushed to master will automatically be incorporated.

Comment (nfosi-service) [reply README.md]: whoops, wrong account :)

**Pull request merged**